recipes/pypardiso/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pypardiso" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c3846870dc2590250980d98568e9dfc99b0271e9e5a9c8714a85cc373e839788
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - "libblas =*=*mkl"
+    - scipy
+    - mkl
+    - mkl-service
+    - psutil
+
+test:
+  imports:
+    - pypardiso
+  requires:
+    - pytest
+    - pip
+  files:
+    - tests/test_basic_solve.py
+    - tests/test_input_A.py
+    - tests/test_scipy_aliases.py
+    - tests/test_input_b.py
+  commands:
+    - pytest -vv tests/
+    - pip check
```

Comment:
@cmutel The builds are failing because of this line, which I think you can get rid of. It basically checks which packages are installed whose dependency is not. I think since `mkl` is installed using `conda`, `pip` fails (I think for `pip check` to succeed, it needs all dependencies installed using `pip`). Anyway, you're already testing `pypardiso` in `pytest -vv tests/`, so if there's a dependency missing, it will be caught during the `import pypardiso`.

Reply:
@conda-forge/help-python Any idea why `pip check` would fail with [missing dependencies](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=339419&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=829) after switching from the Github release to the pypi release? I don't understand hwo things could be [just fine](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=338502&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=780) with everything the same except the location of the downloaded tarball...

Reply:
It's because of `mkl` in https://github.com/haasad/PyPardisoProject/blob/master/setup.py#L7

Reply:
Let's use the github source for now in that case, I'll tidy it up for the next version. Probably my lack of experience with PyPI that caused this.

Reply:
@isuruf Yeah, but no :) `mkl` is installed into the test environment in both the [passing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=338502&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=704) and [failing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=339419&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=752) tests. `pip` is not used to install anything, it is all conda. And pip could find mkl on its path in the passing tests...

The `pip check` suggestion came from someone who knows what they are doing, so I think it is best practice in conda-forge recipes.

@haasad The pypi upload looks fine to me, I assume you did `python setup.py sdist bdist_wheel upload` or used `twine`, there isn't much you could do wrong.

Reply:
There's a pip package called `mkl` which `pip check` is looking for, but we are installing the conda package. So, the best solution is to remove `mkl` in `setup.py` and keep `mkl-service` which has `mkl` as a dependency.

Reply:
@cmutel I released version 0.3.2, where `mkl` is removed as direct dependency in `setup.py`: https://pypi.org/project/pypardiso/

Reply:
@cmutel Could you please update `meta.yaml` to version 0.3.2 (+ the sha256)? :)